test/run.sh

```diff
@@ -62,10 +62,8 @@ if [ ! -f "${RF_VARIABLES}" ]; then
     exit 1
 fi
 
-# shellcheck disable=SC2034
-DEST_DIR="${RF_VENV}" 
```

Comment:
I think if you change this PR to export this variable and add a comment explaining why that is needed it would be sufficient and more future-proof in case we add more dependencies.

Reply:
I agree, done